# This represents a Point with a reference frame, timestamp, and name
Header header
Point point
string name
